lib/Controller/PageController.php

```diff
@@ -44,37 +44,23 @@
 use OCA\Deck\Service\CardService;
 
 class PageController extends Controller {
-	private PermissionService $permissionService;
 	private IInitialStateService $initialState;
-	private ConfigService $configService;
-	private IEventDispatcher $eventDispatcher;
-	private CardMapper $cardMapper;
-	private IURLGenerator $urlGenerator;
-	private CardService $cardService;
-	private IConfig $config;
 
 	public function __construct(
 		string $AppName,
 		IRequest $request,
-		PermissionService $permissionService,
+		private PermissionService $permissionService,
 		IInitialStateService $initialStateService,
```

Comment:
Any reason to not refactor the `initialStateService`  ?

Reply:
Again, it was just about limiting the scope of changes in this PR. The `initialStateService` gets assigned to the `$initialState` in the constructor. I just wasn't sure if that is a specific convention in the NC ecosystem or not.

So basically, I should change all the occurrences of the `$initialState` with the new `$initialStateService` and that will make things cleaner in my opinion. Would you agree with that?